Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -209,6 +209,8 @@ class SearchViewController: SiteTableViewController,
 
     var searchEngines: SearchEngines! {
         didSet {
+            guard let defaultEngine = searchEngines.defaultEngine else { return }
+
```

Comment:
Adding a guard right here will stop everything all the way to the reloading of search engines, is that something you want to do? 

In the previous code, it would still reload search engines

Reply:
Yeah that's fine, since quick search engines takes from the list of ordered engines.